### PR TITLE
2020.01.21/drop python2

### PR DESCRIPTION
**Pull request by mfonism**
This PR drops Python 2 from `django-user-accounts`

[This gist documents the process](https://gist.github.com/mfonism/e12890d34dedc3e4de30efe2d7795184)

### CODE REVIEW

**Comment by KatherineMichel** [comment]
Thank you for doing this and writing up the results @mfonism . It looks great! Something I'm wondering about atm... what about the two instances of `<dictionary>.items()`... do these needs to be changed to... `list(<dictionary>.items())` or is that not the case in this scenario?

https://github.com/pinax/django-user-accounts/blob/5d792e2ac761a3b19d269b62b19f8afa2d30fb1b/account/views.py#L802
https://github.com/pinax/django-user-accounts/blob/5d792e2ac761a3b19d269b62b19f8afa2d30fb1b/account/utils.py#L16

**Comment by mfonism** [comment]
@KatherineMichel thank you for your kind words.

The instances of `<dictionary>.items()` are perfect as they return [view objects](https://docs.python.org/3/library/stdtypes.html#dict-views) which allow us to iterate through the respective dictionaries efficiently.

File: README.rst [hunk 1]
```diff
@@ -63,8 +63,8 @@ Features
 Requirements
 --------------
 
-* Django 1.8, 1.10, 1.11, or 2.0
-* Python 2.7, 3.4, 3.5, or 3.6
+* Django 1.11 or 2.0
+* Python 3.4, 3.5, or 3.6
```
Comment (MattFisher) [inline README.rst]: Should these be updated in line with setup.py? ie
```
* Django 1.11, 2.1, 2.2, or 3.0
* Python 3.6, 3.7, or 3.8
```

Comment (mfonism) [reply README.rst]: Great catch @MattFisher.

I'll update it, thank you.

Comment (MattFisher) [reply README.rst]: Thanks! Looking forward to this patch landing because we want to move to Django 3.0.

**Comment by paltman** [comment]
@mfonism Thanks!

**Pull request merged**